scripts/buildkite/build_tpp.sh

```diff
@@ -62,10 +68,24 @@ then
   exit 1
 fi
 
+echo "Configuring CUDA"
+CUDA_DRIVER=/usr/lib64/libcuda.so.1
+if [ ! -f "${CUDA_DRIVER}" ]; then
+  if [ "${GPU}" ]; then
+    echo "GPU support requires full CUDA driver to be present"
+    exit 1
+  else
+    # When GPU is not used, create link to CUDA stubs to satify dynamic linker.
+    echo "Creating links to CUDA stubs"
+    ln -s ${CUDATOOLKIT_HOME}/lib64/stubs/libcuda.so ${BLD_DIR}/lib/libcuda.so.1
+    ln -s ${BLD_DIR}/lib/libcuda.so.1 ${BLD_DIR}/lib/libcuda.so
```

Comment:
TODO: see if tpp-run can be successfully linked directly with CUDA lib to avoid these runtime hacks

Reply:
What I had in mind didn't work anyway. We'll have to stick with this approach for now.